frontend/src/v5/validation/shared/validators.ts

```diff
@@ -18,7 +18,7 @@ import { formatMessage } from '@/v5/services/intl';
 import { isNull } from 'lodash';
 import * as Yup from 'yup';
 
-export const trimmedString = Yup.string().transform((value) => value.trim());
+export const trimmedString = Yup.string().transform((value) => (value ?? '').trim());
```

Comment:
I'm not sure this should be true. If Yup.string() can accept null/undefined . Then the tranfsorm bit here should do that too instead of converting it to string to then trim it.

It could bring unexpected consequences.